views/welcome.ejs

```diff
@@ -8,16 +8,17 @@
     <p>If you don't want to install your own databases, you can use our databases and support us
     by creating an account for $10. (In Progess)</p>
 
-      <blockquote>
-      <p>MongoDB, a NoSQL Database. <a href="#">MongoDB</a></p>
+      <blockquote class="db-card postgres">
+        <h3 class="db-title">PostgreSQL</h3>
+        <p class="db-discription">A relational database management system emphasizing extensibility and technical standards compliance.</p>
       </blockquote>
-
-      <blockquote>
-      <p>Postgres, an SQL Database. <a href="#">PostGresSQL</a></p>
-      </blockquote> 
-
-      <blockquote>
-      <p>Neo4j, a Graph Database. <a href="#">Neo4j</a></p>
+      <blockquote class="db-card mongodb">
+        <h3 class="db-title">MongoDB</h3>
+        <p class="db-discription">An open source database management system that uses a document-oriented database model which supports various forms of data.<p>
```

Comment:
I think

`An open source database management system that uses a document-oriented database model which supports various forms of data.`

would be better written as

`An open source database management system using a document-oriented database model that supports various forms of data.`

Reply:
Nice